core/themes/bartik/templates/node.tpl.php

```diff
@@ -25,7 +25,7 @@
  *   CSS. The default values can be one or more of the following:
  *   - node: The current template type; for example, "theming hook".
  *   - node-[type]: The current node type. For example, if the node is a
- *     "Post" it would result in "node-post". Note that the machine
+ *     "Article" it would result in "node-article". Note that the machine
```

Comment:
love that we are fixing these, but we should fix them in all core template files so the docs are consistent.